frontend/packages/shared/src/api/prettier.config.js

```diff
@@ -1,6 +1,3 @@
 module.exports = {
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  ...require('../../../../../prettier.config'),
-  printWidth: 999,
```

Comment:
The shared has its own print width, but it should absolutely follow the same rules as rest of the mono-repository. :)

Reply:
This was set to apply to the `api` folder only. It consists only of lists, and they will look quite messy with all the line breaks.

Reply:
I thought so, but I think consistency and same rules overall is better practice? 

If we keep the 999 print-width, then the import statements looks "messy"?

Personally I don't see what's messy with the line breaks? They breaks after "=>" arrow. The same way as they do within regular functions in components and utilities methods.

I can of course add the print width config to 999 again, but then we need to accept a the import statements also become one long single line. ☺️ alternatively see if we can add a custom print-width for import statements.

That's my thought's about this. 😅

Reply:
> If we keep the 999 print-width, then the import statements looks "messy"?

Who reads the import statements? 😛 We could set it to a lower value though, 300 or something, to make it wrap the imports, but not the exports. (One of the things I don't like about Prettier is that it's not very customisable.)

> Personally I don't see what's messy with the line breaks? They breaks after "=>" arrow. The same way as they do within regular functions in components and utilities methods.

It will only wrap the ones that are too long, so they will not look the same. Also, it is easier to sort them when they are all in single lines.